applications/Preprocessor/lib/centaur.cpp

```diff
@@ -325,15 +326,16 @@ void CentaurReader::readPeriodicNodeConnections() {
             // -1 due for Fortran indices
             std::size_t node0 = pairs[2 * i + 0] - 1;
             std::size_t node1 = pairs[2 * i + 1] - 1;
-            // Merge the nodes
-            connectedCoords.unionSets(node0, node1);
+            if (node1 == std::numeric_limits<std::uint32_t>::max()) {
```

Comment:
urgh that is annoying

Reply:
Yes indeed, was quite frustrated by it :(.